docs/src/main/docs/grpc/client/01_introduction.adoc

```diff
@@ -0,0 +1,91 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:pagename: grpc-client-introduction
+:description: Helidon gRPC Client Introduction
+:keywords: helidon, grpc, java
+
+= gRPC Client Introduction
+
+Helidon gRPC Client provides a framework for creating link:http://grpc.io/[gRPC] client applications. The client framework
+allows a uniform way to access gRPC services that use either Protobuf or some custom serialization format. It also allows access
+to gRPC services that use either Java serialization, Protobuf or a custom serialization format.
+
+The class `GrpcServiceClient` acts as the client object for accessing a gRPC service. Creating a `GrpcServiceClient` involves:
+
+1. Creating a `ClientServiceDescriptor` which describes the methods in the service that this client can invoke.
+2. Creating a gRPC `Channel` through which the client communicates with the server.
+
+In later sections in this document, you will see how to customize both `ClientServiceDescriptor` and the `Channel`.
+
+== Quick Start
+
+First, create and run a minimalist `HelloService` gRPC server application as described in the
+https://github.com/aseovic/helidon/blob/master/docs/src/main/docs/grpc/01_introduction.adoc[gRPC Server] documentation.
```

Comment:
This URL should probably be updated to `oracle/helidon` ;)

Reply:
Fixed.